### PR TITLE
new: Updated readme with basic info, badges and default command usage.

### PR DESCRIPTION
**Pull request by rishson**

### CODE REVIEW

**Comment by Tomdye** [comment]
it's `-help`. so `dojo command -help` not `dojo help command`

**Comment by rishson** [comment]
Ohh - good spot.

File: readme.md [hunk 1]
```diff
@@ -1,20 +1,68 @@
...
 
-TODO: Replace with a description of this package
+The CLI is the officially supported way to create and maintain Dojo 2 apps.
 
-## Features
+Its designed to save you time, by promoting a stadardised workflow, and automating away lots of the boring boilerplate tasks.
 
-TODO: Add sections on features of this package
+## Prerequisites
+You will need node V4 or v6+.
```
Comment (Tomdye) [inline readme.md]: i thought we said node v6 for async await?

Comment (rishson) [inline readme.md]: As mentioned yesterday, I'd like to update `engines` at the very least. Ideally, make `4+`, but understand (will put up with) 6+ until `TS 2.1`.

Comment (agubler) [inline readme.md]: Even when TS2.1 comes out it means we will have to emit down to ES5 (currently emits down to ES6).... It comes back to, is it really unreasonable to ask developers to use a node v6+?

@rishson either way please for the sake of us all can you be consistent with your use of `v` and stick with either upper or lower case, not one of each. Kind Regards, my O.C.D.

**Pull request merged**